anaconda.py

```diff
@@ -315,7 +315,7 @@ def setup_environment():
     # see if we're on s390x and if we've got an ssh connection
     uname = os.uname()
     if uname[4] == 's390x':
-        if 'TMUX' not in os.environ and 'ks' not in flags.cmdline and not conf.target.is_image:
+        if 'TMUX' not in os.environ and 'ks' not in flags.cmdline and not conf.target.is_image and not conf.target.is_directory:
```

Comment:
Please, change `not conf.target.is_image and not conf.target.is_directory` to `conf.target.is_hardware`.